libs/api/domains/watson-assistant-chat/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "root": "libs/api/domains/watson-assistant-chat",
+  "sourceRoot": "libs/api/domains/watson-assistant-chat/src",
+  "projectType": "library",
+  "targets": {
+    "lint": {
+      "executor": "@nrwl/linter:eslint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": ["libs/api/domains/watson-assistant-chat/**/*.ts"]
+      }
+    },
+    "test": {
+      "executor": "@nrwl/jest:jest",
+      "outputs": ["coverage/libs/api/domains/watson-assistant-chat"],
+      "options": {
+        "jestConfig": "libs/api/domains/watson-assistant-chat/jest.config.js",
+        "passWithNoTests": true
+      }
+    }
+  },
+  "tags": []
```

Comment:
You need to provide the correct tag for the Nx module boundary rule. So this tag would be `lib:api` so the api app can depend on this library.